CASCEdit/Handlers/DownloadHandler.cs

```diff
@@ -100,7 +114,7 @@ public void AddEntry(CASResult blte)
 			{
 				Hash = blte.Hash,
 				FileSize = blte.CompressedSize - 30,
-				UnknownData = new byte[4],
+				Flags = new byte[Header.NumFlags],
```

Comment:
This is version dependent. Is this correctly initialised/degenerate nicely (i.e. empty) for header.version < 2?

Reply:
Currently it's impossible to create a new entity of this class without using an existing root manifest as a template.